BridgeSDK/BridgeAPI/BridgeAPITypes/MachineGeneratedDoNotEdit/_SBBIdentifierHolder.h

```diff
@@ -33,12 +33,15 @@
 #import <Foundation/Foundation.h>
 #import "SBBBridgeObject.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @protocol _SBBIdentifierHolder
 
 @end
 
 @interface _SBBIdentifierHolder : SBBBridgeObject
 
-@property (nonatomic, strong) NSString* identifier;
+@property (nullable, nonatomic, strong) NSString* identifier;
```

Comment:
ResearchSuite assumes that the identifier is non-null.

Reply:
Good catch--I looked at the ones marked "required" in the Swagger docs and used my judgement on whether they would really always be there, because that flag is for submitting to Bridge, not for what you'll get back. But obviously there were a bunch that should always exist but weren't marked "required" because they're not submitted to the API anywhere (like this one). I will need to revisit this down the road when Alx adds his "x-nullable" flag.